src/ResourceManager/Network/Commands.Network/AvailablePrivateAccessService/AvailablePrivateAccessServiceListMethod.cs

```diff
@@ -0,0 +1,58 @@
+// 
+// Copyright (c) Microsoft and contributors.  All rights reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//   http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// 
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// 
+
+// Warning: This code was generated by a tool.
+// 
+// Changes to this file may cause incorrect behavior and will be lost if the
+// code is regenerated.
+
+using Microsoft.Azure.Commands.Network.Models;
+using Microsoft.Azure.Management.Network;
+using Microsoft.Azure.Management.Network.Models;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Management.Automation;
+using AutoMapper;
+using CNM = Microsoft.Azure.Commands.Network.Models;
+
+namespace Microsoft.Azure.Commands.Network.Automation
+{
+    [Cmdlet(VerbsCommon.Get, "AzureRmVirtualNetworkPrivateAccessService"), OutputType(typeof(List<PSPrivateAccessServiceResult>))]
+    public partial class GetAzureRmAvailablePrivateAccessServiceList : NetworkBaseCmdlet
+    {
+        [Parameter(
+            Mandatory = true,
+            HelpMessage = "The location.",
+            ValueFromPipelineByPropertyName = true)]
+        [ValidateNotNullOrEmpty]
+        public string Location { get; set; }
+
+        public override void Execute()
+        {
+            base.Execute();
+
+            var vAvailablePrivateAccessServiceList = this.NetworkClient.NetworkManagementClient.AvailablePrivateAccessServices.List(Location);
+            List<PSPrivateAccessServiceResult> psAvailablePrivateAccess = new List<PSPrivateAccessServiceResult>();
+            foreach (var vAvailablePrivateAccessService in vAvailablePrivateAccessServiceList)
+            {
+                psAvailablePrivateAccess.Add(Mapper.Map<CNM.PSPrivateAccessServiceResult>(vAvailablePrivateAccessService));
+            }
+            WriteObject(psAvailablePrivateAccess);
```

Comment:
@EvgenyAgafonchikov this should be `WriteObject(psAvailablePrivateAccess, true)` to enumerate the collection of objects in the pipeline